tests/unit/Core/Upload/StreamableUploaderTest.php

```diff
@@ -147,4 +147,31 @@ public function testThrowsExceptionWithFailedUpload()
 
         $uploader->upload();
     }
+
+    public function testLastChunkSendsCorrectHeaders()
+    {
+        $resumeUriResponse = new Response(200, ['Location' => 'theResumeUri']);
+        $response = new Response(200, ['Location' => 'theResumeUri'], $this->successBody);
+
+        $this->requestWrapper->send(
+            Argument::which('getMethod', 'POST'),
+            Argument::type('array')
+        )->willReturn($resumeUriResponse);
+
+        $this->requestWrapper->send(
+            Argument::that(function($request) {
+                return $request->getHeaderLine('Content-Length') == '10';
+            }),
+            Argument::type('array')
+        )->willReturn($response);
```

Comment:
We can. It fails without the mock, but explicitly adding the invocation expectation will be better.